go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/kubernetes-sigs/kernel-module-management v1.1.0
 	github.com/onsi/ginkgo/v2 v2.13.1
 	github.com/onsi/gomega v1.30.0
+	github.com/rh-ecosystem-edge/kernel-module-management v0.0.0-20231024110254-591541ea5a45
```

Comment:
It's a bit annoying, but could you also update ContainerFileNotice to reflect changes in go.mod?